pyproject.toml

```diff
@@ -44,16 +44,7 @@ importlib-metadata = { version = ">=0.12", python = "<3.8" }
 pytest = "*"
 pytest-randomly = "*"
 pytest-cov = "*"
-
-# Linting
-isort = "*"
-black = { version = "*", allow-prereleases = true }
-flake8 = "*"
-flake8-bugbear = "*"
-flake8-builtins = "*"
-flake8-comprehensions = "*"
-mypy = "*"
```

Comment:
Just a note regarding the earlier comment about mypy in CI: we may have to add mypy back here.

Reply:
👍